model/notification/registry.go

```diff
@@ -0,0 +1,66 @@
+package notification
+
+import (
+	"fmt"
+
+	"github.com/evergreen-ci/evergreen/model/event"
+	"github.com/mongodb/grip"
+	"github.com/pkg/errors"
+	"gopkg.in/mgo.v2/bson"
+)
+
+var triggerRegistry map[string][]trigger = map[string][]trigger{}
+
+func init() {
+	triggerRegistry[event.ResourceTypeTest] = []trigger{testTrigger}
+}
+
+func getTriggers(resourceType string) []trigger {
+	triggers, ok := triggerRegistry[resourceType]
+	if !ok {
+		return nil
+	}
+
+	return triggers
+}
+
+func testTrigger(e *event.EventLogEntry) ([]Notification, error) {
```

Comment:
This is being used by the EventMetaJob tests in units package, so I can't move it to a test. 

The only reason this exists is because I wanted an end to end test for event drive notifications, but events have been implemented yet. I'm planning on deleting this test trigger in the same commit that implements the first event, so I don't want to do any engineering around allowing special events/triggers to be inserted just for tests